pkg/docker/docker.go

```diff
@@ -69,8 +66,8 @@ const (
 	// timeout is reached, certain Docker API calls might error out.
 	DefaultDockerTimeout = 60 * time.Second
 
-	// DefaultShmSize is the default shared memory size to use (in bytes) if not specified.
-	DefaultShmSize = int64(1024 * 1024 * 64)
+	// defaultShmSize is the default shared memory size to use (in bytes) if not specified.
+	defaultShmSize = int64(1024 * 1024 * 64)
```

Comment:
ok, i'll defer, but if I catch that we had to make it public in the future, you own me a beer :-)